evm/src/cpu/push0.rs

```diff
@@ -0,0 +1,32 @@
+use plonky2::field::extension::Extendable;
+use plonky2::field::packed::PackedField;
+use plonky2::hash::hash_types::RichField;
+use plonky2::iop::ext_target::ExtensionTarget;
+
+use crate::constraint_consumer::{ConstraintConsumer, RecursiveConstraintConsumer};
+use crate::cpu::columns::CpuColumnsView;
+use crate::cpu::membus::NUM_GP_CHANNELS;
+
+pub fn eval_packed<P: PackedField>(
+    lv: &CpuColumnsView<P>,
+    yield_constr: &mut ConstraintConsumer<P>,
+) {
+    let filter = lv.op.push0;
+    let push_value = lv.mem_channels[NUM_GP_CHANNELS - 1].value;
+    for &limb in &push_value[0..] {
```

Comment:
nit: Why not `for &limb in &push_value {` ?

Reply:
lol good catch!! i copy-pasted this code and just thoughtlessly changed `1..` to `0..`…